docker-compose.yml

```diff
@@ -7,6 +7,8 @@ services:
       DOMAIN:
       API_WEBSOCKET_PORT:
       ROOM_STORE_DIR: /var/ttbud/rooms
+    volumes:
```

Comment:
I don't think this is ideal. Sharing production and dev rooms seems scary. Can you make this volume work like the other production volumes?

Reply:
Fixed the line of code you wrote because you had a problem with it with your solution... Why is this my PR, again?

Reply:
because you knew I'm too lazy to do it